src/components/SidePanel/sidePanel.module.css

```diff
@@ -1,30 +1,24 @@
-.favorites {
+.sidePanel {
     height: 30vh;
     width: 15vw;
     position: relative;
     left: 0;
     top: 0;
     padding-top: 1rem;
-    background-color: slategray;
+    background-color: var(--primary-accent);
```

Comment:
Semantically speaking, primary-accent is not a variable for background colors unless you're matching the actual accent. if this was on purpose it's fine, just make sure you understand the concept of accents vs other types of colors